card/mod/01_core/spec/set/all/fetch_spec.rb

```diff
@@ -157,6 +174,25 @@
         expect(card.new_record?).to be_falsey
       end
     end
+
+    describe 'default_content option' do
+      context "when card doesn't exist" do
+        it 'initializes card with default content' do
+          card = Card.fetch 'non-existent',
+                            new: { default_content: 'default content' }
+          expect(card.content).to eq 'default content'
+        end
+      end
+      context "when new card exist" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.